src/AppBundle/Resources/views/Firm/show.json.twig

```diff
@@ -0,0 +1,36 @@
+            
+{
+
+   "@context": "http://schema.org",
+   "@type": "Organization",
+   
+   "url": "{{ url('firm_show', {'id': firm.id}) }}",
+   
+   "name": "{{ firm.name }}",
+   "address": "{{ firm.streetAddress }}",
+   
+   "location": {
+   "@type": "Place",
+   "url": ""http://sws.geonames.org/{{ firm.city.geonameid }}/about.rdf""
+   },
```

Comment:
This comma will cause problems for any firm that is missing startDate, endDate, and titleFirmRoles.